docs/sources/helm-charts/mimir-distributed/configure/configure-redis-cache.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configure Redis cache"
+menuTitle: "Redis cache"
+description: "Learn how to configure Grafana Mimir to use external Redis as cache"
+---
+
+# Configure Redis cache
+
+Beside built-in support for Memcached, Mimir also supports external Redis for chunks-cache, index-cache, results-cache and metadata-cache. To use Redis, we must disable the build-in Memcached
```

Comment:
[nit] Is there such thing as internal Redis?

```suggestion
Beside built-in support for Memcached, Mimir also supports Redis for the chunks-cache, index-cache, results-cache and metadata-cache. To use Redis, we must disable the build-in Memcached
```

Reply:
The intention was to highlight the user need to install Redis themselves "external" from mimir-distributed helm chart. But the nit comment makes sense.